wapitiCore/config/attacks/users.txt

```diff
@@ -0,0 +1,45 @@
+admin
```

Comment:
Is there a special order for this? otherwise use alphabetical order

Reply:
fixed